client/Styles/form.scss

```diff
@@ -0,0 +1,144 @@
+.sticky-form-wrap {
```

Comment:
Any and all styling criticism is very much appreciated!

Reply:
kind of a mixture of my own stuff plus taking exact measurements from wework site

Reply:
looks pretty good so far but noticed alot of repeated css property values. id recommend declaring sass variables